locations/spiders/nedbank_mz.py

```diff
@@ -0,0 +1,108 @@
+from urllib.parse import urlencode
+
+from scrapy import Spider
+from scrapy.http import Request
+
+from locations.categories import Categories, apply_category
+from locations.hours import DAYS_EN, DAYS_PT, DELIMITERS_EN, DELIMITERS_PT, OpeningHours
+from locations.items import Feature, merge_items
+from locations.spiders.nedbank_za import NEDBANK_SHARED_ATTRIBUTES
+
+
+class NedbankMZSpider(Spider):
+    name = "nedbank_mz"
+    item_attributes = NEDBANK_SHARED_ATTRIBUTES
+    start_urls = [
+        "https://www.nedbank.co.mz/en/customer-support/branches.aspx",
+        "https://www.nedbank.co.mz/apoio-ao-cliente/rede-de-balc%C3%B5es.aspx",
+    ]
+    saved_items = {"en": {}, "pt": {}}
```

Comment:
I think I like the general idea of merging, but I dislike the idea of keeping all the items in memory before emitting them.

In this case, each language has its own page? Could we request one page, stuff all the details in `meta` on a request to the other language and then output one result?

Reply:
I see what you mean and this wasn't how I first thought of doing it. I agree that would be better in general, but not so easy here.

Here we have six results on a page, but the order of results isn't the same between en and pt. So maybe a result is on page 2 in en, but page 3 in pt. Meaning we need to get all en results before trying any matching with pt results.

How about we hold all en results, then put them in meta to request the pt results and output results as they match from there?

Reply:
After thinking through this a bit more, I'm ok with doing this in limited cases. Let's keep it as is.